django_amazon_ses/backends/boto.py

```diff
@@ -18,23 +18,28 @@ class EmailBackend(BaseEmailBackend):
     Attributes:
         conn: A client connection for Amazon SES.
     """
-    def __init__(self, region_name=None, fail_silently=False, **kwargs):
+    def __init__(self, fail_silently=False, **kwargs):
         """Creates a client for the Amazon SES API.
 
         Args:
-            region_name: Amazon region for SES endpoint.
             fail_silently: Flag that determines whether Amazon SES
                 client errors should throw an exception.
 
         """
         super(EmailBackend, self).__init__(fail_silently=fail_silently)
-        if region_name is None:
-            region_name = getattr(
-                settings,
-                'DJANGO_AMAZON_SES_REGION',
-                'us-east-1'
-            )
-        self.conn = boto3.client('ses', region_name=region_name)
+        access_key_id = getattr(settings, 'AWS_ACCESS_KEY_ID', None)
+        secret_access_key = getattr(settings, 'AWS_SECRET_ACCESS_KEY', None)
+        region_name = getattr(
+            settings,
+            'DJANGO_AMAZON_SES_REGION',
```

Comment:
Given the introduction of `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`, do you think it makes sense to rename this to `AWS_DEFAULT_REGION`? (That's the name of the environment variable Boto looks for when trying to determine the region to target.)

Reply:
Makes sense to me. I've made the suggested change in the latest revision and included a release note.